src/pages/Login/Login.jsx

```diff
@@ -1,12 +1,20 @@
-import React from 'react'; 
-import { Link } from "react-router-dom";
+import React from 'react'
+import Button from '../../ui/Button/Button'
+import Logo from '../../images/pc-eyes.png'
 
-const Login = () => (
-  <div>
-    <h1>Login</h1>
 
-    <Link to="/">Home</Link>
+import './Login.scss'
+
+const Login = ({onClick}) => (
```

Comment:
We don't need to receive this here. This way, we'll need to pass it through the route and we don't have a reason to do this there. You can create a function inside this component just for now or just pass an empty function (I'll show bellow)